homeassistant/components/notify/__init__.py

```diff
@@ -32,6 +34,13 @@
 
 SERVICE_NOTIFY = "notify"
 
+NOTIFY_SERVICE_SCHEMA = vol.Schema({
+    vol.Required(ATTR_MESSAGE): cv.template,
+    vol.Optional(ATTR_TITLE, default=ATTR_TITLE_DEFAULT): cv.string,
```

Comment:
I would love to get rid of the `ATTR_TITLE_DEFAULT` and leave it up to platforms to populate it if necessary. (but that might be out of scope for this PR)